cmake/modules/BuildArrow.cmake

```diff
@@ -49,6 +49,7 @@ function(build_arrow)
     find_package(gRPC REQUIRED)
     find_package(Protobuf REQUIRED)
     find_package(c-ares 1.13.0 QUIET REQUIRED)
+    # find_package(utf8proc 2.2.0 REQUIRED) # BOOST already sets this up
```

Comment:
When I uncomment this, I get an error due to a naming conflict of Boost already setting this up.